index.html

```diff
@@ -1,12 +1,25 @@
 <!DOCTYPE html>
-  <html lang="en">
-  <head>
-     <meta charset="UTF-8">
-     <title>Angular ToDo App</title>
-     <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.2/angular.min.js"></script>
-     <script src="app.js"></script>
-  </head>
-  <body>
-    <h1>To Do!</h1>
-  </body>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Winnie's To-Do List</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.2/angular.min.js"></script>
+  <script src="app.js"></script>
+  <script src="data.js"></script>
+  <link rel="stylesheet"href="style.css">
+</head>
+<body data-ng-app="myTodo">
+  <h1>Winnie's To-Do List</h1>
+  <div class="list" data-ng-controller="vm">
+    <div ng-repeat="task in todos">
+      <input ng-model="task.completed" type="checkbox"/>
+      <label ng-hide="edit" class="done-{{task.completed}}"> {{task.name}} </label>
```

Comment:
Interesting! Did this actually work? Angular has [ngClass](https://docs.angularjs.org/api/ng/directive/ngClass) for this sort of situation.